modules.yml

```
@@ -215,6 +223,10 @@ categories:
         slug: 'micronaut-email'
         title: 'Micronaut Email'
         description: 'Integration with email providers such as Amazon Simple Email Service, Postmark, Mailjet or SendGrid.'
+      grpc:
+        slug: 'micronaut-grpc'
+        title: 'Micronaut GRPC'
+        description: 'Integration between Micronaut and GRPC.'
```

```suggestion
        description: 'Integration with GRPC.'
```

From the [trademark guidelines](https://micronaut.io/brand-guidelines/): 

> You may not use the word Micronaut as a verb or a noun. It should be used only as an adjective followed by a noun (e.g., “I love using the Micronaut® framework,” not “I love using Micronaut!”).